README.md

```diff
@@ -95,7 +95,7 @@ You can install NewPipe using one of the following methods:
 
 We recommend method 1 for most users. APKs installed using method 1 or 2 are compatible with each other, but not with those installed using method 3. This is due to the same signing key (ours) being used for 1 and 2, but a different signing key (F-Droid's) being used for 3. Building a debug APK using method 4 excludes a key entirely. Signing keys help ensure that a user isn't tricked into installing a malicious update to an app.
 
-In the meanwhile, if you want to switch sources for some reason (e.g. NewPipe's core functionality was broken and F-Droid doesn't have the update yet), we recommend following this procedure:
+In the meanwhile, if you want to switch sources for some reason (e.g. NewPipe's core functionality was broken and F-Droid didn't have the update for quite some weeks), we recommend following this procedure:
```

Comment:
```suggestion
In the meanwhile, if you want to switch sources for some reason (e.g. say NewPipe's core functionality brakes and F-Droid doesn't have the update yet), we recommend following this procedure:
```

Reply:
how about this? that explicitly states the hypothetical-ity of the example.

Reply:
I would say that the "e.g." already expresses that it's just a hypothetical example. If you decide to change it though, it should be "breaks", not "brakes".

Reply:
Also, I think "yet" was fine; no need for "quite some weeks."

Reply:
yeah, i have made it to be "yet" in the new suggestion. i hope u checked it. :)

Reply:
"e.g. say" is redundant. Use only one of them. I think "e.g." will be more universally understood. Also "latest update".
```suggestion
In the meanwhile, if you want to switch sources for some reason (e.g. NewPipe's core functionality breaks and F-Droid doesn't have the latest update yet), we recommend following this procedure:
```